src/cascadia/TerminalSettingsModel/ActionMap.h

```diff
@@ -0,0 +1,118 @@
+/*++
+Copyright (c) Microsoft Corporation
+Licensed under the MIT license.
+
+Module Name:
+- ActionMap.h
+
+Abstract:
+- A mapping of key chords to actions. Includes (de)serialization logic.
+
+Author(s):
+- Carlos Zamora - September 2020
+
+--*/
+
+#pragma once
+
+#include "ActionMap.g.h"
+#include "IInheritable.h"
+#include "Command.h"
+#include "../inc/cppwinrt_utils.h"
+
+// fwdecl unittest classes
+namespace SettingsModelLocalTests
+{
+    class KeyBindingsTests;
+    class DeserializationTests;
+    class TerminalSettingsTests;
+}
+
+namespace winrt::Microsoft::Terminal::Settings::Model::implementation
+{
+    typedef size_t InternalActionID;

```

Comment:
```suggestion
    using InternalActionID = size_t;
```

typedef is what we said when we did not know how to speak C++

C++11 came along and allowed us something a little more . . . elegant.